packages/amplify-util-mock/src/__tests__/api/lambda-invoke.test.ts

```diff
@@ -53,6 +53,7 @@ describe('Invoke local lambda function', () => {
     expect(printer.info).toBeCalledWith(JSON.stringify(echoInput, undefined, 2));
     expect(printer.error).toBeCalledTimes(0);
     expect(printer.info).toBeCalledWith('Finished execution.');
+    expect(isBuilt).toBeTruthy();
```

Comment:
nit: although it doesn't matter in this context, I prefer to use `.toBe(true)` over `.toBeTruthy()` because it's more strict and avoids common errors when checking returned promises

```suggestion
    expect(isBuilt).toBe(true);
```